lib/event_bridge.rb

```diff
@@ -0,0 +1,31 @@
+class EventBridge
+
+  def self.put_event(event_name, payload, rule)
+    raise StandardError.new("EventBridge put failed due to missing source prefix") if rule["PREFIX"].empty?
+    raise StandardError.new("EventBridge put failed due to empty event bus name") if rule["EVENTBRIDGE"].empty?
+
+    client = Aws::EventBridge::Client.new(
+      credentials: Aws::InstanceProfileCredentials.new(),
+      region: ENV["AWS_REGION"] || "ap-southeast-2"
+    )
+
+    resp = client.put_events({
+      entries: [
+        {
+          time: Time.now,
+          source: "#{rule["PREFIX"]}.shift72.com",
```

Comment:
I had hoped the source would be the service name we use. "Content_", putting our domain in this doesn't seem right.

Reply:
Currently splitting on the `_` so it would just be `source: "Content.shift72.com"`.  Maybe I should downcase that too.

Reply:
yes to downcase :)